website/README.md

```diff
@@ -24,6 +24,9 @@ If you are not familiar with Jekyll, be sure to check out its
 [documentation][jekyll-docs]. It is recommended to follow the provided
 step-by-step tutorial.
 
+On top of Jekyll, we use the [just-the-docs][just-the-docs] theme, having
```

Comment:
Hmm, this flows a bit odd IMO, the "having" part of the sentence doesn't get me. Little proposal, not sure if a lot better:

> On top of Jekyll, we use the [just-the-docs] theme, which itself also includes several configuration options.

Reply:
Looks good!

Reply:
Done.